Modules/Loadable/VolumeRendering/MRML/vtkMRMLVolumePropertyNode.h

```diff
@@ -206,6 +211,11 @@ class VTK_SLICER_VOLUMERENDERING_MODULE_MRML_EXPORT vtkMRMLVolumePropertyNode
   /// independent value-wise, and they do not have third and fourth axes.
   double EffectiveRange[2];
 
+  /// MRML node ID of the selected preset volume property node
+  /// Important! The node with this ID belongs to the presets private scene in volume rendering
+  ///            logic, and not the slicer MRML scene this node is in!
+  char* PresetVolumePropertyNodeID;
```

Comment:
In this case, I think it would be better to store a "preset name" string instead, which would be matched with the node name of the volume property preset.

Reply:
Maybe less confusing, but I see it as an unnecessary conversion back and forth, because it's the ID that is used.

Reply:
It is more than confusing, thinking a bit more about it, it may actually lead to incorrect behavior if we use node IDs.

Node IDs in the private scene are assigned in undeterministic way, because extensions can register their own presets and random order. But even for built-in presets, node IDs may change if we add/remove/reorder them.

Also, if you store something like 'vtkMRMLPropertyNode1' in the scene, it is very uninformative, compared to seeing "CTCardiac1" instead. This case is different from other node references, as we cannot look up what 'vtkMRMLPropertyNode1' means, as it is not in the scene file.